### PR TITLE
Add deprecation warning to 'data.lena()'

### PR DESCRIPTION
**Pull request by sergiopasra**
This is a first step towards fixing #1855 by adding a deprecation warning to `data.lena()`

### CODE REVIEW

File: skimage/data/__init__.py [hunk 1]
```diff
@@ -56,6 +57,7 @@ def camera():
     return load("camera.png")
 
 
+@deprecated()
```
Comment (JDWarner) [inline skimage/data/__init__.py]: If this is called with a string argument, the user gets an even more helpful message about which function they should transition to. I'd recommend 

`@deprecated('skimage.data.astronaut()')`

Comment (stefanv) [inline skimage/data/__init__.py]:
```
@deprecated('skimage.data.astronaut')
```

Comment (JDWarner) [inline skimage/data/__init__.py]: Use @stefanv's version ;)

**Comment by sergiopasra** [comment]
If everyone agrees, I will put ' astronaut'

**Comment by sergiopasra** [comment]
Ahh, typo in the commit message, but no in the code

**Comment by JDWarner** [comment]
You can amend the commit message with 

`git commit --amend` (two dashes before amend)

and then to push, you have to use


`git push -f [remote] [branch]` (single dash before f)

to push changed history.

**Comment by sergiopasra** [comment]
@JDWarner Thanks, it's done

**Comment by OrkoHunter** [comment]
@stefanv @sergiopasra @JDWarner I would like to work on rewriting the tests which uses the deprecated image. I am new to the community and want to start from here !

**Comment by stefanv** [comment]
@OrkoHunter That would be excellent.  We'd have to look at the tests one by one to see how reliant they are on lena itself.  E.g., if the test merely uses lena as input, but doesn't check any values, we can probably get away with swapping in another image.  But if it checks feature positions, e.g., we'd have to recalibrate on another image.  Pick one, start working on it, make a "WIP" (work in progress) pull request (i.e., just name the pull request something like "WIP: Remove lena from xyz test"), and when you get stuck we'll give you a hand.  Thanks for contributing to scikit-image!

**Pull request merged**